subprojects/docs/src/docs/userguide/jvm/groovy_plugin.adoc

```diff
@@ -202,7 +202,10 @@ The Groovy plugin also modifies some source set properties:
 [[sec:groovyCompile]]
 == GroovyCompile
 
-The Groovy plugin adds a link:{groovyDslPath}/org.gradle.api.tasks.compile.GroovyCompile.html[GroovyCompile] task for each source set in the project. The task type extends the `JavaCompile` task (see <<building_java_projects.adoc#sec:compile, the relevant Java Plugin section>>). The `GroovyCompile` task supports most configuration options of the official Groovy compiler.
+The Groovy plugin adds a link:{groovyDslPath}/org.gradle.api.tasks.compile.GroovyCompile.html[GroovyCompile] task for each source set in the project.
+The task type extends the `JavaCompile` task (see <<building_java_projects.adoc#sec:compile, the relevant Java Plugin section>>).
```

Comment:
```suggestion
The task type extends the `AbstractCompile` task (see <<building_java_projects.adoc#sec:compile, the relevant Java Plugin section>>).
```

Reply:
Thanks for that, did a slightly different fix in the end.